de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v2_0/editor/FSKEditorJSNodeModel.java

```diff
@@ -524,10 +531,55 @@ protected PortObject[] performExecute(PortObject[] inObjects, ExecutionContext e
 //      }
 
       // Collect R packages
-      final Set<String> librariesSet = new HashSet<>();
-      librariesSet.addAll(new RScript(modelScript).getLibraries());
-      librariesSet.addAll(new RScript(visualizationScript).getLibraries());
-      packages = new ArrayList<>(librariesSet);
+      if(metadata != null && SwaggerUtil.getLanguageWrittenIn(metadata) != null) {
+        String languageWrittenIn = SwaggerUtil.getLanguageWrittenIn(metadata);
+        String pythonPath = null; // Path to the Python executable in the conda environment
+        String envPath = null; // Path to the conda environment
+        if(PreferencePage.getPython3CondaEnvironmentDirectoryPath() != null)
+          envPath = PreferenceInitializer.getPython3Env();
+        else
+          envPath = PreferenceInitializer.getPython2Env();
+        
+        if(((FskPortObject)inObjects[0]).overridePackages) {
```

Comment:
Throws an error if the Editor has no input
<img width="119" alt="image" src="https://github.com/SiLeBAT/FSK-Lab/assets/38949263/f1529219-393c-4126-9ea3-00e74476d44b">

`ERROR FSK Editor JS        3:1        Execute failed: Cannot read field "overridePackages" because "inObjects[0]" is null
`

Reply:
Another thing: the python scripts are stored in the user folder. We should either put them in the workflow folder or use the KNIME python-integration way to send commands to the python server instead of directly calling a python process.